editor.planx.uk/src/@planx/components/Send/Public.tsx

```diff
@@ -61,10 +67,9 @@ const CreateSendEvents: React.FC<Props> = ({
   ]);
 
   // Send makes a single request to create scheduled events in Hasura, then those events make the actual submission requests with retries etc
-  const url = `${
-    import.meta.env.VITE_APP_API_URL
-  }/create-send-events/${sessionId}`;
-  const request: any = useAsync(async () => {
```

Comment:
This `any` was the cause of a bug where event ids are not logged to the breadcrumb.

The list below has `request.value.bops`, `request.value.uniform` etc, but it should be `request.value.data.bops`, `request.value.data.uniform` etc